py/occampy3/wrappers/report.py

```diff
@@ -8,3 +19,18 @@ def __init__(self, ref):
         :param: ref: the reference to the Report object returned from the CPP engine
         """
         self._ref = ref  # type Report_cpp
+
+    def set_separator(self, separator: SeparatorType) -> None:
+        self._ref.setSeparator(separator.value)

```

Comment:
This will prevent passing in just an integer, so we'll have to keep that in mind.